lib/utilities.js

```diff
@@ -77,4 +89,4 @@ async function startWebInspectorService (udid, socket) {
   }
 }
 
-export { getConnectedDevices, getOSVersion, startLockdownSession, startSyslogService, startWebInspectorService, connectPort };
+export { getConnectedDevices, getOSVersion, getDeviceTime, startLockdownSession, startSyslogService, startWebInspectorService, connectPort };
```

Comment:
this line is too long

Reply:
I think I will start creating this `service` object we were talking about last week. It is getting too long indeed